docs/rebase.md

```diff
@@ -83,6 +95,10 @@ When running this command, you will be prompted for the branch on top of which y
 
 If any merge conflicts occur while rebasing, the state of the operation will be reflected in the dashboard, with successful rebased commits showing a check mark, and the commit with the current merge conflict showing an X as well as a list of files with conflicts.
 
+#### Toggle perserve merges mode (`m`)
+
+In default, merges will be lost when rebasing. If perserve merges mode is on, commits within a merge are consolidated in a single item. If any conflicts occur while rebasing a merge commit, the commits within the merge will be shown and the conflicting commit will be marked.
```

Comment:
Your could edit the merge message of a merge commit in the current code.  
I argue that if you really need to edit a commit of a merged branch, you should use the non-preserving mode.

Manipulating commits within a merge is tricky, particularly if there are nested merges.
Although, editing commit messages within a merge can be made possible, but it will be a tedious job.